### PR TITLE
[FIX] mail: chatter composer adding scroll with newline

### PR DESCRIPTION
**Pull request by dhp-odoo**
**PURPOSE**
Reproduction Steps (It is reproducing in Firefox only):
    - In Chatter
    - Type text in the first line of the composer to fill it completely
    - Then press "Enter"
    - A scrollbar will appear (before we reach max-height),
      even though we are supposed to have auto-height adjust.

**SPECIFICATION**
We are now assigning the height as 0px first to the textarea and then
it will reassigned based on the content height resolves the issue.

Task : 2463635

### CODE REVIEW

File: addons/mail/static/src/components/composer_text_input/composer_text_input.js [hunk 1]
```diff
@@ -185,6 +185,9 @@ class ComposerTextInput extends Component {
      */
     _updateHeight() {
         this._mirroredTextareaRef.el.value = this.composer.textInputContent;
+        // In firefox, if you fill the first line of the composer completely and hit "enter"
+        // the scrollbar will appear (before we reach max-height)
+        this._textareaRef.el.style.height = "0px";
```
Comment (seb-odoo) [inline addons/mail/static/src/components/composer_text_input/composer_text_input.js:190]: Hello, thank you for the PR;

This appear to fix the scroll issue correctly, but this would revert the change that was done in 52f3db8ef61d4284d8ea624022bc1355f7f8cf00 and it willreintroduce the other issue.
If you test in community with a chatter that is below the form, and you press enter many times, you will see the whole page scroll is reset.

Do you see any other way to resolve it?

Comment (dhp-odoo) [reply addons/mail/static/src/components/composer_text_input/composer_text_input.js:190]: @seb-odoo . Actually with my changes actually I unable to reproduce the issue. Can you please elaborate more on this?

And I can not find any other way specific to firefox, as any of the changes regarding firefox will affect other browsers too.

**Comment by seb-odoo** [comment]
task is not reachable and v14 is almost obsolete
